modules/migration-rolling-back-migration-manually.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc
+// * migration_toolkit_for_containers/troubleshooting-mtc
+
+[id="migration-rolling-back-migration-manually_{context}"]
+= Rolling back a migration manually
+
+When a migration fails, certain resources are left behind for debugging. You can roll back a migration manually by deleting the migration resources and unquiescing the application.
+
+[NOTE]
+====
+Rollback is not required if the application was not stopped during migration because the original application is still running on the source cluster.
+====
+
+The following resources might be left after a failed migration:
+
+* Indirect migration: `stage` pods (source and destination clusters)
+* Direct volume migration:
+** Config maps (source and destination clusters)
+** `Secret` CRs (source and destination clusters)
+** `Rsync` CRs (source cluster)
+** `Service` CRs (destination cluster)
+** `Route` CRs (destination cluster)
+
+If you later run the same migration plan successfully, the resources from the failed migration are deleted automatically.
+
+.Procedure
+
+. For indirect migration, delete the `stage` pods:
+.. Delete the `stage` pods on all clusters:
++
+[source,terminal]
+----
+$ oc delete $(oc get pods -l migration.openshift.io/is-stage-pod -n <namespace>) <1>
+----
+<1> Namespaces specified in the `MigPlan` CR.
+
+. For direct volume migration, delete the migration resources:
+
+.. Get the resources on all clusters:
++
+[source,terminal]
+----
+$ oc get all -n <namespace> <1>
+----
+<1> Namespaces specified in the `MigPlan` CR.
+
+.. Delete the following resources on the source cluster:
+* Config maps
+* `Secret` CRs
+* `Rsync` CRs
+
+.. Delete the following resources on the destination cluster:
+* Config maps
+* `Secret` CRs
+* `Rsync` CRs
+* `Service` CRs
+* `Route` CRs
+
```

Comment:
@XinRedhat 
If the user deletes the target namespace, do I still need to include the step to delete the resources in the target namespace?